README.md

```diff
@@ -1,4 +1,45 @@
-# ewasm testnet coordination / documentation repo
+# Ewasm public testnet
+
+Welcome to the Ewasm public testnet! This repository is the primary point of coordination for the testnet. Read on for more information about how to:
+
+- Transact on the testnet, such as by deploying smart contracts compiled to Ewasm bytecode
+- Add a node to the testnet
+- Participate in mining
+- Participate in network forks
+- Contribute to development and devops
+
+## Background
+
+Ewasm, which stands for Ethereum-flavored WebAssembly, is the primary candidate to replace EVM (the Ethereum virtual machine) as part of the Ethereum 2.0 "Shasper" roadmap. Ewasm is a deterministic smart contract execution engine built on the modern, standard [WebAssembly virtual machine](https://webassembly.org/).
```

Comment:
Perhaps add to the shasper thing: and it is also proposed for adoption on the mainnet.